stacks_queues/stack.py

```diff
@@ -21,18 +21,26 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store.length == 0:
+            raise StackEmptyException
+        else:
+            return self.store.remove_last()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.length() == 0
 
     def __str__(self):
```

Comment:
👀 This works, but you might take advantage of the `__str__` method in LinkedList